### ISSUE TITLE
QF 4.16 doesn't validate if no default identity is defined

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#10]
If an account has no default identity it should log this in the Javascript error console. Instead the validation fails silently and the following error is shown:

```
Error in QuickFolders.Options.decryptLicense():

logDebug is not defined  quickfolders-register.js:552
```

**Comment by RealRaven2000** [comment]
It seems the lines 551 - 553 are missing the prefix "util." for the logDebug function:

```
           if (ac.incomingServer.username != "nobody") {
            util.logDebug("Account " + ac.incomingServer.prettyName + " has no default identity!");
          }

```
Test version with quick fix:
[QuickFolders-wx-4.17pre21.zip](https://github.com/RealRaven2000/QuickFolders/files/3811013/QuickFolders-wx-4.17pre21.zip)

**Comment by RealRaven2000** [comment]
Fixed in 4.17, released on 04/11/2019
